austrakka/components/fieldtype/value/__init__.py

```diff
@@ -12,7 +12,7 @@
 @click.pass_context
 def value(ctx):
     """Commands related to fieldtype values"""
-    ctx.creds = ctx.parent.creds
+    ctx.context = ctx.parent.context
```

Comment:
Why is this sort of assignment necessary? The information is already there.

Reply:
Because otherwise we need to know at what level command is nested when we access this information. eg. for a subcommand it would be `ctx.parent.parent.context` and a sub-subcommand should be `ctx.parent.parent.parent.context`. It's stupid but that's how click seems to do it